torch/utils/data/dataset.py

```diff
@@ -73,6 +73,10 @@ def __len__(self):
         return self.cumulative_sizes[-1]
 
     def __getitem__(self, idx):
+        if idx < 0:
+            if -idx > len(self):
+             raise ValueError("absolute value of index should not exceed dataset length")
```

Comment:
linter is failing here (indentation is not a multiple of 4).
Could you look into fixing it?

Reply:
Oh weird - sorry about that. Fixed.